### ISSUE TITLE
Minehut Forums and Panel

### ISSUE THREAD

**Issue by ItzOP** [Minehut/Meta#275]
### Feature request

**Checklist**
- [x] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [x] This is not a bug report, support request or plugin request
- [x] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [x] This issue does not contain any sensitive information 

**Platform**

Website

**Describe the feature**

One of the first things you learn, when taking any sort, of course, related to website development is consistency. Alongside a few other bits. Over my time on Minehut, the forums have gone through many updates as well as the panel.
But it's never been consistent with each other. Both the panel and forums always seem to use different colours.
I admire the hard work being put into updating Minehut features recently.
But every time I go to the forums it feels like a completely different company, it looks nothing like the panel.
The forums use this darker blue colour, that I haven't really seen before. The default theme (only theme at the moment) is white whereas on the panel it's dark.

So essentially, what I'm suggesting is make it feel like the same site. It feels odd at the moment.
But of course, this is just my opinion and I'd like to hear what others think.

**How the feature is useful**

The feature alone isn't extremely useful or in any way needs to be prioritised.
It's one of the jobs that would just be a nice touch, and in no way does it need to be rushed anytime soon.
Especially considering you just did a forums update, I don't expect Minehut to go out of their way to do another just yet.

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by MiginsMC** [comment]
What exactly are you suggesting?

**Comment by jellz** [comment]
The forum update wasn't facilitated by Minehut, it was an update from their upstream provider (Invision). Before the update, there was a dark theme, so the dark theme will probably return soon. The forum and panel won't look similar because the panel uses Material Design and the forum doesn't. Do you have any other suggestions on how to make them look more connected, aside from the missing dark/light themes?

**Comment by ItzOP** [comment]
> The forum update wasn't facilitated by Minehut, it was an update from their upstream provider (Invision). Before the update, there was a dark theme, so the dark theme will probably return soon. The forum and panel won't look similar because the panel uses Material Design and the forum doesn't. Do you have any other suggestions on how to make them look more connected, aside from the missing dark/light themes?

No, I understand that. But, even before this update. It wasn't consistent. In my opinion, it still didn't feel the same as the panel.
It should feel like a seamless transition like it's the same website. But before and after the update it still doesn't.

**Comment by ItzOP** [comment]
The main things to make them look similar would just be similar header/footer, similar colours. There seems to be weird dark blue colour, that isn't the normal one on the new forums as well.

**Comment by drtshock** [comment]
Yeah design within the invision environment has never been our forte. We definitely need to do a design update on the forums, especially since the new update broke our old theme.
